docs/en/your-profile-and-preferences/log-out-of-status.md

```diff
@@ -0,0 +1,29 @@
+---
+id: 387
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Log out of Status
+
+When you decide not to use Status on the current device, you can log out to keep your profile secure. Your profile automatically logs out if you uninstall Status or sometimes after you update Status.
```

Comment:
Thoughts:

- When the app logs the user out, the action is on the app and not on the user.
- Commonly, the user isn't the one updating the app. This occurs automatically.

Considering these two, I would write something like this:

> Status logs you out automatically after some app updates or when you uninstall the app.